pkg/manifests/manifests.go

```diff
@@ -3586,3 +3737,42 @@ func trustedCABundleVolume(configMapName, volumeName string) v1.Volume {
 		},
 	}
 }
+
+func getAlertmanagerTLSSecretsName(alertmanagerConfigs []AdditionalAlertmanagerConfig) []string {
```

Comment:
(nit) secrets also include bearer tokens.

```suggestion
func getAdditionalAlertmanagerSecrets(alertmanagerConfigs []AdditionalAlertmanagerConfig) []string {
```